oak-jackrabbit-api/src/main/java/org/apache/jackrabbit/api/JackrabbitSession.java

```diff
@@ -252,4 +258,12 @@
      */
     Node getNodeOrNull(final String absPath) throws RepositoryException;
 
+    /**
+     * Returns the set of principals associated with this session.
+     *
+     * @return the associated principals (may be the empty set if no principal are associated).
+     * @throws RepositoryException If an error occurs.
+     * @since 1.40.0
```

Comment:
The since refers to the bundle version I guess, but this looks weird as the other method refer to Jackrabbit 2 versioning 2.x.. while this refers to Oak versioning. That leads to the case that 2.11.1 is older than 1.40.0...